CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `tt install tt` returns expected exit status code on unsuccessful dependency check.
 - `tt pack`: failed to start instances using systemctl due to `permissions denied`.
 - `tt uninstall tt`: does not work with `x.y.z` version format.
+- `tt pack`: work impossible combination of flags `with-integrity-check` and `wihtout-binaries`.
```

Comment:
It looks like a `tt-ee` change.

Reply:
Deleted from this added to `tt-ee`. Also added test for this change.